### PR TITLE
docs: Fix references to section list templates

### PR DESCRIPTION
**Pull request by wholien**
fixes issue [#2032] (https://github.com/spf13/hugo/issues/2032)

### CODE REVIEW

File: docs/content/content/types.md [hunk 1]
```diff
@@ -60,6 +60,9 @@ Create a directory with the name of the type in `/layouts`. Type is always singu
 ### Create single template
 Create a file called `single.html` inside your directory. *E.g. `/layouts/post/single.html`*.
 
+### Create list template
+Create a file called `post.html` inside your directory. *E.g. `/layouts/section/post.html`*.
```
Comment (moorereason) [inline docs/content/content/types.md]: s/your/the section lists template/

Comment (wholien) [inline docs/content/content/types.md]: @moorereason what do you mean? According to the docs [here] (https://gohugo.io/templates/list/) it should be `/layouts/section/YOURSECTION.html`. Should I add something else or change it to be more clear?

Comment (moorereason) [inline docs/content/content/types.md]: I mean, I don't like the `inside your directory` part.  In the surrounding context, "your directory" is `/layouts/SECTION`.  It should say something like `inside the section lists template directory, "/layouts/section"`.

Comment (wholien) [inline docs/content/content/types.md]: @moorereason fixed. Can you review it. Should I squash these two commits?

**Comment by moorereason** [comment]
Looks better.  You can squash now.

**Comment by wholien** [comment]
@moorereason squash

**Comment by wholien** [comment]
@moorereason can I merge? I guess only you guys have write access?

**Comment by moorereason** [comment]
Merged in 16ad628 and 405de5a (v0.15.docs).

Thanks!

**Pull request closed without merging**